### DISCUSSION TITLE
コンフィグファイルが参照できない

### DISCUSSION THREAD

**Discussion by mamimumemo700** [espnet/espnet#4829] [Q&A]
自前のデータセットを用意してASRモデルの学習をしようとしています。
run.sh
` 
set -e
set -u
set -o pipefail

./asr.sh \
    --stage 3 \
    --stop_stage 13 \
    --lang en \
    --asr_config conf/train_asr_transformer \
    --inference_config conf/decode_asr \
    --lm_config conf/train_lm \
    --token_type bpe \
    --nbpe  30\
    --audio_format wav \
    --train_set train_nodev \
    --valid_set train_dev \
    --asr_tag "transfer_learning_with_pre_trained_model" \
    --pretrained_model "../../an4/asr1/exp/asr_train_asr_transformer_raw_en_bpe30_sp/latest.pth" \
    --test_sets "train_dev test" \
    --bpe_train_text "data/train_nodev/text" \
    --lm_train_text "data/train_nodev/text" "$@"`
 　
でコンフィグファイルの指定をしているのですがどれも
lm_train.py: error: No such file : conf/train_lm.yaml
asr_train.py: error: No such file : conf/train_asr_transformer.yaml
とのことで読み込めていないようです。ディレクトリ構造はTEMPLATEのものそのままですしconfの中にそれぞれのファイルはきちんとあります。上記オプションの指定無しならレシピは最後まで動きますし、an4などのレシピを試したときにはこのエラーは出ませんでした。
何が原因として考えられるでしょうか？

**Comment by kan-bayashi** [comment]
おそらく参照しているコンフィグがsymlinkで、symlinkの参照先が存在しないのではないでしょうか？
